src/async_wrap.cc

```diff
@@ -856,6 +856,17 @@ void AsyncWrap::EmitDestroy(Environment* env, double async_id) {
     env->SetImmediate(&DestroyAsyncIdsCallback, CallbackFlags::kUnrefed);
   }
 
+  // If the list gets very large empty it faster using a Microtask.
+  // Microtasks can't be added in GC context therefore we use an
+  // interrupt to get this Microtask scheduled as fast as possible.
+  if (env->destroy_async_id_list()->size() == 16384) {
+    env->RequestInterrupt([](Environment* env) {
+      env->isolate()->EnqueueMicrotask(
+        reinterpret_cast<v8::MicrotaskCallback>(DestroyAsyncIdsCallback),
```

Comment:
```suggestion
        [](void* arg) {
          DestroyAsyncIdsCallback(static_cast<Environment*>(arg));
        },
```

(Just a suggestion, I think this might be a bit cleaner/easier to grok?)

Reply:
done